react/components/ProductImages/styles.css

```diff
@@ -14,12 +14,6 @@
   height: auto;
 }
 
-.carouselGaleryThumbs
-  :global(.swiper-slide):not(:global(.swiper-slide-thumb-active))
-  .carouselThumbBorder {
-  border-width: 0px;
-}
-
```

Comment:
did this affect something? Is this safe?

Reply:
It is because of this https://github.com/vtex-apps/store-components/pull/653/files#r351917610